docs/metrics/bertscore.md

```diff
@@ -0,0 +1,117 @@
+# BERTScore
+BERTScore is a metric used in NLP workflows to measure textual similarity between machine-generated text and reference text. Unlike [BLEU](bleu.md), it leverages pretrained BERT embeddings to capture the semantic and contextual information of words and phrases in both the generated and reference texts. This approach makes BERTScore more effective at assessing the quality of generated text because it considers not only exact word matches but also the overall meaning, fluency, and order of the output.
```

Comment:
Just some suggestions:
`between machine-generated text and reference text` => `between any two texts`
I don't think we should assume that the other text is always generated `generated and reference` `generated text` - it could be 2 just two randomly made texts.

`unlike BLEU` => `unlike BLEU, ROUGE, and traditional text similarity metrics`